pom.xml

```diff
@@ -54,6 +54,18 @@
 				<artifactId>hamcrest</artifactId>
 				<version>${hamcrest.version}</version>
 			</dependency>
+      <dependency>
+			  <groupId>junit</groupId>
+			  <artifactId>junit</artifactId>
+			  <version>4.12</version>
+			  <scope>test</scope>
```

Comment:
why adding junit 4 when there is always junit 5 (junit jupiter)?

Reply:
Applied